OpenRA.Mods.Common/Activities/CaptureActor.cs

```diff
@@ -30,6 +30,12 @@ public CaptureActor(Actor self, in Target target, Color? targetLineColor)
 
 		protected override void TickInner(Actor self, in Target target, bool targetIsDeadOrHiddenActor)
 		{
+			if (manager == null)
```

Comment:
`manager` still can't be null, because line 28 does not return null (but would throw if the trait does not exist).

Reply:
I see, didn't realize that. Fixed in last commit.